src/basic_recipes/legend.jl

```diff
@@ -34,10 +36,10 @@ creates a legend from a colormap
 """
 @recipe(ColorLegend, colormap, colorrange) do scene
     Theme(
-        width = (20, 200),
+        width = (20, lift(x -> x[2], theme(scene, :resolution))),
```

Comment:
should rather use `lift(x-> width(x)[2], pixelarea(scene))`.. The theme value won't update when resizing the window

Reply:
```suggestion
        width = (20, lift(x -> x[2], pixelarea(scene))),